addons/account/models/account_move.py

```diff
@@ -2714,7 +2733,8 @@ class AccountMoveLine(models.Model):
         domain="[('deprecated', '=', False), ('company_id', '=', company_id)]",
         check_company=True,
         tracking=True)
-    account_internal_type = fields.Selection(related='account_id.user_type_id.type', string="Internal Type", store=True, readonly=True)
+    account_internal_type = fields.Selection(related='account_id.user_type_id.type', string="Internal Type", readonly=True)
```

Comment:
Because this means we can't change the type of a used account for performance reasons. See the commit message https://github.com/odoo/odoo/pull/48651/commits/8e7dd05cab62f18a5ee9c8fcee6c96119ef24e48